library/aes.c

```diff
@@ -1442,7 +1454,10 @@ int mbedtls_internal_aes_decrypt( mbedtls_aes_context *ctx,
     if( flow_control == tindex + dummy_rounds + 8 )
     {
         /* Validate control path due possible fault injection */
-        return 0;
+        if( output_dup == output && input_dup == input )
+        {
+            return 0;
+        }
```

Comment:
I think we should clean data in case of MBEDTLS_ERR_PLATFORM_FAULT_DETECTED error

Reply:
This is a part of this PR: https://github.com/ARMmbed/mbedtls/pull/3481

Reply:
@AndrzejKurek , Sorry, I thought we have only one PR for both issues now